### PR TITLE
PMM-7463 Test DbaaS PXC Operator V1.7.0

### PR DESCRIPTION
**Pull request by beatahandzelova**

### CODE REVIEW

**Comment by beatahandzelova** [comment]
Could not add reviewers so Jiri Ctvrtka requested review.

File: docs/setting-up/server/dbaas.md [hunk 1]
```diff
@@ -140,10 +140,10 @@ alias kubectl='minikube kubectl --'
     PMM_PASS_B64="$(echo -n "${PMM_PASS}" | base64)";
 
     # Install the PXC operator
-    curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/bundle.yaml \
+    curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/v1.7.0/deploy/bundle.yaml \
     | kubectl apply -f -
-    curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/secrets.yaml \
-    | sed "s/pmmserver:.*=/pmmserver: ${PMM_PASS_B64}/g" \
+    curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/v1.7.0/deploy/secrets.yaml \
+    | sed "s/pmmserver:.*=/pmmserver: ${PMM_PASS}/g" \
```
Comment (BupycHuk) [inline docs/setting-up/server/dbaas.md:146]: so we don't need to use base64-ed passwords in secrets anymore, right?

Comment (beatahandzelova) [reply docs/setting-up/server/dbaas.md:146]: Yes, more details here https://perconacorp.slack.com/archives/C01JQ9F1EAF/p1613550951130200?thread_ts=1613486613.121300&cid=C01JQ9F1EAF

Comment (BupycHuk) [reply docs/setting-up/server/dbaas.md:146]: Thank you

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**